src/js/components/Form/stories/Select.js

```diff
@@ -16,16 +24,22 @@ const FormFieldSelect = () => {
   return (
     <Grommet theme={grommet}>
       <Box align="center" pad="large">
-        <Form value={value} onChange={onChange}>
+        <Form
+          value={value}
+          onChange={onChange}
+          onSubmit={() => console.log('Submit', value)}
+        >
           <FormField label="Label" name="select">
             <Select
+              multiple
```

Comment:
Since I think the 80% use case for Select is not `multiple`. I think we shouldn't make this change.

Reply:
Done.